webpack/stories/docs/APIMiddlewareUsage.md

```diff
@@ -0,0 +1,137 @@
+# Example: how to use the API middleware
+
+The example API returns a JSON object like this:
+```json
+{
+  "items": [
+    {"id": 319, "title": "setting", "action": "update"},
+    {"id": 150, "title": "bookmark", "action": "create"}
+  ],
+}
+```
+
+```js
+  /** MyComponent.js*/
+
+  class MyComponent extends React.Component {
```

Comment:
Can it be done using a functional component?
Instead `componentDidMount` you can use `useEffect` in the `index` file.